docs/README.md

```diff
@@ -25,9 +25,21 @@ $ pip install -U -r requirements/docs.txt
 $ make -C . clean && make -C . html
 ```
 
+To generate the documentation without running the examples:
+
+```bash
+$ make -C . clean && make -C . html-no-examples
+```
 #### Under Windows
 
 ```bash
 $ ./make.bat clean
 $ ./make.bat html
+```
+
+To generate the documentation without running the examples:
+
+```bash
+$ ./make.bat clean
+$ ./make.bat html-no-examples
```

Comment:
commands starting with `./` are actually not recognised by windows(cmd). `make.bat`  or `make` works though. for e.g `make clean` or `make.bat clean`.